.env.example

```
@@ -0,0 +1,3 @@
+AUTH_SECRET={AUTH_SECRET}
+AUTH_GOOGLE_ID={AUTH_GOOGLE_ID}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_GOOGLE_ID key should go before the AUTH_SECRET key

(UnorderedKey)